focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -55,19 +56,22 @@ end
 
 --[[
   -- method figures out the orientation the photo was shot at by looking at the metadata
-  -- returns 90, 270, or 0
+  -- returns 90, 270, or 0 (in trigonometric sense)
 --]]
 function DefaultDelegates.getShotOrientation(photo, metaData)
   local dimens = photo:getFormattedMetadata("dimensions")
   local orgPhotoW, orgPhotoH = parseDimens(dimens) -- original dimension before any cropping
 
   local metaOrientation = ExifUtils.findValue(metaData, "Orientation")
-  if (string.match(metaOrientation, "90") and orgPhotoW < orgPhotoH) then
+  if (string.match(metaOrientation, "90 CW") or string.match(metaOrientation, "90")) and orgPhotoW < orgPhotoH then
+    return 270
```

Comment:
correct. Should have used .find for example. You want me to commit the correction ?